esp/templates/customforms/index.html

```diff
@@ -156,7 +159,19 @@ <h3><a href='#'>Links</a></h3>
 					<p>Pick -
 					<select id="links_id_pick">
 					</select>
-					</p>		
+					</p>
+
+                    <p>User type -
+					<select id="links_id_tl">
+                        <option value="learn">Students</option>
+                        <option value="teach">Teachers</option>
+					</select>
+					</p>
+
+                    <p>Module -
```

Comment:
I assume the lack of entries here means I have neither customforms nor quizzes enabled, but it would be helpful to have help text show up if that's the case saying "For teachers to be able to fill out this form for this program, you need to enable one of the following modules: ... "

![image](https://user-images.githubusercontent.com/25392075/117383577-1765f600-aeaf-11eb-90f8-63cef57a9908.png)